python_dashing/server/static/js/components/BarChart.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {Component, PropTypes} from 'react';
+import styles from './Number.css';
+import WidgetBox from './WidgetBox.jsx';
+import {Bar} from 'react-chartjs';
+
+const BarChart = ({data, title='Bar Chart'}) =>
+  <WidgetBox color="#3498db" className={styles.container}>
+    <h1 className={styles.heading}>{title}</h1>
+    <Bar data={data} width="600" height="250"/>
```

Comment:
On the initial render, the data is not loaded yet and the `resolve` call returns `undefined`.